api/pkg/tests/fixtures.go

```diff
@@ -0,0 +1,33 @@
+package test
+
+import (
+	"context"
+	"encoding/json"
+	"github.com/aws/aws-sdk-go-v2/config"
+	"github.com/aws/aws-sdk-go-v2/service/s3"
+	"github.com/danielgtaylor/huma/v2/humatest"
+	"github.com/stretchr/testify/require"
+	"happenedapi/pkg/images"
+	"happenedapi/pkg/server"
+	"io"
+	"testing"
+)
+
+func MakeTestServer(t *testing.T) humatest.TestAPI {
```

Comment:
Create test fixture for creating a test server connected to real S3. Possibly use localstack in the future or run this only in a staging environment.